File: 프로그래머스/1/87389. 나머지가 1이 되는 수 찾기/나머지가 1이 되는 수 찾기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int n) {
    int answer = 0;
    for(int i = 2; i < n ; i++){
        if(n%i == 1){
            answer = i;
            // 최소값이므로 구하면 바로 루프탈출
            break;
        }
    }
    return answer;
}

File: 프로그래머스/1/87389. 나머지가 1이 되는 수 찾기/나머지가 1이 되는 수 찾기.py
def solution(n):
    answer = 0
    for i in range(2, n):
        if n % i == 1:
            answer = i
            # 최소값이므로 구하면 바로 루프 탈출
            break
    return answer

# 예제 사용
result = solution(10)
print(result)
